rust-runtime/aws-smithy-types/Cargo.toml

```diff
@@ -13,9 +13,10 @@ repository = "https://github.com/smithy-lang/smithy-rs"
 [features]
 byte-stream-poll-next = []
 http-body-0-4-x = ["dep:http-body-0-4"]
+http-body-1-x = ["dep:http-body-1-0", "dep:http-body-util", "http-body-0-4-x"]
```

Comment:
If this pulls in `http-body-0-4-x`, then it will have to pull that in forever since removing it would technically break the API. It should pull in the dependency instead so that the APIs associated with the feature don't come with it.

Reply:
good point—these things were required to actually compile this in all cases but I'll add private APIs to fix this up.